specifyweb/frontend/js_src/lib/components/WebLinks/spec.ts

```diff
@@ -161,48 +161,29 @@ function parseField(item: RawWebLink, part: string): ParsedWebLink {
       type: 'FormattedResource',
       formatter: field.title,
     };
-  return {
-    type: 'PromptField',
-    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-    label: localized(field?.title || field?.name || part),
-  };
+  return { type: 'UrlPart', value: localized('') };
```

Comment:
nice fallback 👍
this should behave as if user gave an empty response to a prompt field, which would exactly match the behavior before sp7 supported this feature